README.md

```diff
@@ -66,7 +66,7 @@ keyring::key_get("springer", keyring = "scythe")
 ## Acknowledgments
 Work on this package was supported by:
 
-- NSF-PLR grant #1546024 to M. B. Jones, S. Baker-Yeboah, J. Dozier, M. Schildhauer, and A. Budden
+- NSF-PLR grant #2042102 to M. B. Jones, A. Budden, M. Schildhauer, and J. Dozier
```

Comment:
Thanks. Please don't remove the 2015 grant, just add the new one to the top of the list. They both supported it.